libs/shared/utils/src/lib/utils.ts

```diff
@@ -80,3 +82,18 @@ export function withTimeout<T>(asyncFn: () => Promise<T>, timeoutMs: number) {
       });
   });
 }
+
+export function killProcessTree(pid: number, signal = 'SIGTERM') {
+  if (process.platform !== 'win32') {
+    try {
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      require('tree-kill')(pid, signal);
```

Comment:
Note that `tree-kill` already uses `taskkill` for Windows. Should maybe look into inlining the function like we did for Nx since that package hasn't been maintained in a long time.

Reply:
sure, I can do that.